midi_app_controller/gui/binds_editor.py

```diff
@@ -176,6 +281,28 @@ def __init__(
 
         self.setLayout(layout)
 
+        self.flashing_knobs = set([])
+        self.thread_list = []
```

Comment:
It is not used anywhere. Maybe create a method that waits for all the threads and call it on exit in `BindsEditor`.